e2e_tests/provider_cfg/all/sqlite-kim-all-providers.toml

```diff
@@ -0,0 +1,53 @@
+[core_settings]
+# The CI already timestamps the logs
+log_timestamp = false
+log_error_details = true
+
+# The container runs the Parsec service as root, so make sure we disable root
+# checks.
+allow_root = true
+
+[listener]
+listener_type = "DomainSocket"
+timeout = 200 # in milliseconds
+socket_path = "/tmp/parsec.sock"
+
+[authenticator]
+auth_type = "Direct"
+admins = [ { name = "list_clients test" }, { name = "1000" }, { name = "client1" }, { name = "spiffe://example.org/parsec-client-1" } ]
+#workload_endpoint="unix:///tmp/agent.sock"
+
+[[key_manager]]
+name = "sqlite-manager"
+manager_type = "SQLite"
+database_path = "./kim-mappings/sqlite/sqlite-key-info-manager.sqlite3"
+
+[[provider]]
+provider_type = "MbedCrypto"
+key_info_manager = "sqlite-manager"
+
+[[provider]]
+provider_type = "Tpm"
+key_info_manager = "sqlite-manager"
+tcti = "mssim"
+owner_hierarchy_auth = "tpm_pass"
+
+[[provider]]
+provider_type = "Pkcs11"
+key_info_manager = "sqlite-manager"
+library_path = "/usr/local/lib/softhsm/libsofthsm2.so"
+user_pin = "123456"
+# The slot_number mandatory field is going to replace the following line with a valid number
+# slot_number
+
+[[provider]]
+provider_type = "CryptoAuthLib"
+key_info_manager = "sqlite-manager"
+device_type = "always-success"
+iface_type = "test-interface"
```

Comment:
Don't know about the discussions around the `trusted-services` provider, but was that added into the `all-providers` group in the end? If so, should it be added into here?

Reply:
Yeah, will add it

Reply:
Ah, I remember now what the problem was: the Trusted Services build we use for testing on the CI transforms the whole TS stack into a library that you link into the executable. Because the crypto Trusted Service is built on top of an implementation of Mbed Crypto, if you try to link both that and the implementation used for the Mbed Crypto provider you end up with:
1) some sort of symbol aliasing, so both the TS provider and the Mbed Crypto provider call the same methods (which we don't want, because they use different versions of the Mbed library, so you get bugs/crashes when stuff gets called with wrong params)
2) because of the way Mbed Crypto stores keys, they end up sharing all their keys on disk. This is a problem because both providers assume that namespace is "free", so they both try to start with a key with ID `1`, and then throw errors when it's not possible

There might've been more problems involved, but for now we're just keeping the Mbed Crypto provider enabled here.